api/v1beta1/cryostat_types.go

```diff
@@ -92,6 +94,17 @@ type StorageConfiguration struct {
 	PVC *PersistentVolumeClaimConfig `json:"pvc,omitempty"`
 }
 
+// ReportConfiguration is used to determine how many replicas of cryostat-reports
+// the operator should create and what the resource limits of those containers
+// should be. If no replicas are created then Cryostat is configured to use basic
+// subprocess report generation. If at least one replica is created then Cryostat
+// is configured to use remote report generation, pointed at a load balancer service
+// in front of the cryostat-reports replicas.
+type ReportConfiguration struct {
+	Replicas                    int32 `json:"replicas,omitempty"`
+	corev1.ResourceRequirements `json:",inline"`
```

Comment:
Could you add a comment for Replicas? It will be shown to the user when creating a CR using the form view. Also I think some OLM Descriptors make sense here:
- [urn:alm:descriptor:com.tectonic.ui:podCount](https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md#1-podcount)
- [urn:alm:descriptor:com.tectonic.ui:resourceRequirements](https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md#2-resourcerequirements)